db/migrate/20221123185757_create_users.rb

```diff
@@ -0,0 +1,13 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.text :name
+      t.string :bio
+      t.string :photo
+      t.string :postscounter
+      t.string :integer
```

Comment:
- [ ] I think it would be better if you update the migration file for creating the **users** table to match data from the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) as required for the project. e.g.: it is better `name` to of type `string`, `bio` to be `text`, `posts_counter` to be an `integer`, and remove the `integer` column as it is not supposed to be there.

Once you update the above changes kindly make sure you update the `schema.rd` file. You can run this command to do that: 
```
 rails db:drop db:create db:migrate
```

Reply:
@tresorsawasawa,
 I chose to try and actually alter the table rather than dropping it and creating it again,
and I have learned so much from it